scripts/lib/api/generateToc.ts

```diff
@@ -39,7 +39,32 @@ export function generateToc(
   pkg: Pkg,
   results: Array<{ meta: Metadata; url: string }>,
```

Comment:
Consider either defining a type `MetadataWithUrl` or using `HtmlToMdResultWithUrl` from `HtmlToMdResult.ts`. The downside of `HtmlToMdResultWithUrl` is it makes your test setup more complex.

Reply:
Good idea! I used `HtmlToMdResultWithUrl` to match the type we were already passing from `conversionPipeline.ts`